config/initializers/pagy.rb

```diff
@@ -0,0 +1 @@
+# Pagy initializer file (3.7.5)
```

Comment:
shouldn't we have this content instead? https://github.com/ddnexus/pagy/blob/master/lib/config/pagy.rb

Reply:
Yes, I remove the content because Rubocop was complaining about some lines.
If you think it's worth I can add it and exclude the file

Reply:
I'd ignore rubocop on this file yes 👍